app/utils/geolocation.js

```
@@ -0,0 +1,55 @@
+export const getUserCountry = async () => {
+  try {
+    // Try to get from localStorage first
+    const storedCountry = localStorage.getItem(COUNTRY_KEY);
+    if (storedCountry) {
+      return storedCountry;
+    }
+
+    // Use ipapi.co for geolocation (free, no API key needed)
+    const response = await fetch('https://ipapi.co/json/');
+    const data = await response.json();
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)